challenges.yaml

```diff
@@ -126,7 +126,7 @@
       Extract all IP addreses from files that
       that start with "access.log" printing one
       IP address per line.
-    example: find . -name "access.log*" -exec grep -Po '^\d+\.\d+\.\d+\.\d+' {} \;
+    example: grep -Po '^\d+\.\d+\.\d+\.\d+' **/access.log*
```

Comment:
Ah, oops; I meant to add the `-h` option, which makes grep not show the filenames.